zproject/test_extra_settings.py

```diff
@@ -51,6 +51,9 @@
 if PUPPETEER_TESTS:
     # Disable search pills prototype for production use
     SEARCH_PILLS_ENABLED = False
+    # Disable hotspots during puppeteer tests.
+    TUTORIAL_ENABLED = False
```

Comment:
We have disabled hotspots during puppeteer tests because it will send `NON_INTRO_HOTSPOT` due to the `TUTORIAL_FINISHED` check is [moved below after we send the non-intro hotspot](https://github.com/zulip/zulip/pull/17557/commits/aa1976ea6c7535c699c217275b6e7de57b42c9c6#diff-37b1b8cd541dd2f2830f3ffeddfd29df77efb005849fe6fd2277e8dcd608604aL78).

If we send the `intro_draft` hotspot then the puppeteer test fails when it will try to mock the save to draft behavior.